bot/functions/joinAutorole.ts

```diff
@@ -0,0 +1,9 @@
+import { Client, Snowflake } from "discord.js";
+
+export function setupAutorole(client: Client, roleId: Snowflake){
```

Comment:
jesli to jest domyslna rola ktora raczej nie bedzie zmieniana (chyba ze w przyszlosci sie zdecydujemy dac taki interfejs w bocie) to dałbym to roleId w osobny plik config i nazwał coś typu KSI_MEMBER_ROLE_ID

Reply:
Mozna tam albo dac po prostu ID roli albo jesli bedziemy chcieli ja modyfikowac z bota to wtedy jako zmienna

Reply:
to wtedy jakiś storage będzie potrzebny na bocie, to póki co mysle ze zmiana default roli uzytkownikow bedzie zbedna to wyrzucilbym to do configa zeby potem miec wspolna zmienna w calym bocie

Reply:
to po tym jeszcze tylko zrebasowac sie na nowe pliki z mastera i bedzie done

Reply:
config.ts jest na mainie jak coś juz gotowy to tylko dodac